src/components/Organisms/PageHeader/PageHeader.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import { Button } from '@/components/Atoms/Button'
+import { Divider } from '@/components/Atoms/Divider'
+import { Picture } from '@/components/Atoms/Picture'
+import { LanguageDropdown } from '@/components/Molecules/LanguageDropDown'
+import { PageHeaderProps } from './PageHeader.interface'
+import { StyledContainer, StyledDropdownContainer } from './PageHeader.styled'
+
+export const PageHeader: React.FC<PageHeaderProps> = ({
+  imageURL,
+  languageOptions,
+  onChange,
+  onClick,
+}) => {
```

Comment:
@philjonasfiltered What does this return?? Would be nice to see it...
```suggestion
}): JSX.Element => {
```